content/en/load-testing/Manual/800-master-controller-cheatsheet.md

```diff
@@ -8,19 +8,35 @@ type: docs
 description: >
   You can find a list of available commands below.
 ---
+## Usage
+
+- `mastercontroller [<other options>]` runs in [interactive mode](../310-test-execution/#interactive-mode). Any command can be chosen to be executed next.
+- `mastercontroller --c` or `mastercontroller --commands` displays all the available options and a brief guide about their usage.
+- `mastercontroller -auto [<other options>]` runs a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands). by executing all commands automatically.
+
+### Example(Windows PowerShell)
+
+- This would display all the available command options
+```powershell 
+C:\xlt-5.7.1\bin> ./mastercontroller.cmd --c 
+```
+- This would make a load test run in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands), executing commands automatically in combination with [embedded mode](../310-test-execution/#embedded-mode).
+```powershell 
+C:\xlt-5.7.1\bin> ./mastercontroller.cmd -auto -embedded -comment "Test Run" 
+```
 
 ## List of Command Options
-1. `-auto` 
-2. `-c` or `--commands`
-3. `-comment`
-4. `-D`
-5. `-embedded`
-6. `-faf`
-7. `-noDownload`
-8. `-o`
-9. `--only-download`
-10. `-pf`
-11. `report`
-12. `sequential`
-13. `testPropertiesFile`
-14. `timezone`
+- `-auto` is to be [used](../310-test-execution/#auto-mode) for running a load test in [non-interactive mode](../310-test-execution/#non-interactive-mode-scripted-commands).
```

Comment:
Might be misleading to link to both. The `-auto` option is starting mastercontroller in auto mode (https://xltdoc.xceptance.com/load-testing/manual/310-test-execution/#auto-mode), which is also non-interactive but you have no chance to define the commands that should be run, it just uses the default command sequence.

Reply:
Alright, should I be removing the link for auto-mode?